.flowconfig

```diff
@@ -1,6 +1,8 @@
 [ignore]
 .*/node_modules/.*/test/.*\.json$
 .*/node_modules/fbjs/.*
+.*/node_modules/react-native/.*
```

Comment:
React-swipeable-views, but it only gets pulled in if you import the native version. Not ideal being on the build server but it doesn't actually get used or pulled into the web version of this package.